DEVGUIDE.md

```diff
@@ -123,39 +127,15 @@ For **Release** this corresponds to these steps, which you can run individually
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable7 /p:Configuration=Release
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable78 /p:Configuration=Release
     msbuild src/fsharp-library-unittests-build.proj /p:TargetFramework=portable259 /p:Configuration=Release
-    msbuild VisualFSharp.sln /p:Configuration=Release
-    msbuild vsintegration\fsharp-vsintegration-unittests-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-src-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-project-templates-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-item-templates-build.proj /p:Configuration=Release
+    msbuild vsintegration/fsharp-vsintegration-deployment-build.proj /p:Configuration=Release
+    msbuild vsintegration\fsharp-vsintegration-unittests-build.proj  /p:Configuration=Release
+    msbuild tests/fsharp/FSharp.Tests.fsproj /p:Configuration=Release
     src\update.cmd release -ngen
     tests\BuildTestTools.cmd release 
 
-### 4. [Optional] Install the Visual F# IDE Tools and Clobber the F# SDK on the machine
```

Comment:
No replacement. This PR removes the special 9055 version redirection, and uses one unified assembly version. VS developers run the VSIX they want to debug, and VS will load all the correct assemblies at runtime. I'll update the batch file to make this more clearer and send a PR out.